### ISSUE TITLE
Incorrectly failing while parsing SVG class attribute

### ISSUE THREAD

**Issue by darovic** [htmllint/htmllint#208]
There appears to be an issue parsing an SVG tag with multiple classes assigned in a single class attribute.

For example, Bootstrap's documentation fails with error
 `_gh_pages/index.html: line 171, col 32, attribute names must match the format: dash`

The HTML source code being referenced is 
```html
...
    <svg class="text-primary mb-2" viewBox="0 0 32 32" width="32" height="32" fill="none" stroke="currentcolor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2">
      <path d="M9 22 C0 23 1 12 9 13 6 2 23 2 22 10 32 7 32 23 23 22 M11 26 L16 30 21 26 M16 16 L16 30" />
    </svg>
...
```
It appears to fail at the dash in `mb-2`. 

This appears limited to SVG tags, as similar class declarations elsewhere pass without issue (eg. `<div class="col-12 col-md-4 p-3 p-md-5 bg-light border border-white">` passes in the same file).

Not a breaking issue, but thought I'd report it. Hope this helps in the future.

**Comment by mlochbaum** [comment]
It's failing on the property `viewBox`, which doesn't match the `dash` style, that is, lowercase and dash-separated. Making it possible to set different options for svgs and other elements which tend to break from html standards is a priority for htmllint; see #137 which mentions `viewBox` specifically.

The line and column numbers are reported incorrectly in 0.6.0; this was fixed by 47e6d8d which will be included in the next release (definitely this week).
